.devcontainer/devcontainer.json

```diff
@@ -27,24 +28,32 @@
                 },
                 "isort.args": ["--profile", "black"],
                 "python.analysis.autoImportCompletions": true,
-                "python.defaultInterpreterPath": "/usr/local/conda/envs/Recommenders/bin/python",
+                // Conda env name *must* align with ENV_HOME in Dockerfle
+                "python.defaultInterpreterPath": "/root/conda/envs/Recommenders/bin/python",
                 "python.testing.pytestEnabled": true,
-                // set the directory where all tests are 
+                // Test directory 
                 "python.testing.pytestArgs": ["tests"]
             },
-            // Add the IDs of extensions you want installed when the container is created.
+            // VS Code extensions to install on container create
             "extensions": [
-                "ms-python.black-formatter",  // https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter
-                "ms-python.isort",  // https://marketplace.visualstudio.com/items?itemName=ms-python.isort
-                "ms-python.mypy-type-checker",  // https://marketplace.visualstudio.com/items?itemName=ms-python.mypy-type-checker
-                "ms-python.pylint",  // https://marketplace.visualstudio.com/items?itemName=ms-python.pylint
-                "ms-python.python",  // https://marketplace.visualstudio.com/items?itemName=ms-python.python
-                "ms-toolsai.datawrangler",  // https://marketplace.visualstudio.com/items?itemName=ms-toolsai.datawrangler
-                "ms-toolsai.jupyter"  // https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter
+                // https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter
+                "ms-python.black-formatter",
+                // https://marketplace.visualstudio.com/items?itemName=ms-python.isort
+                "ms-python.isort",
+                // https://marketplace.visualstudio.com/items?itemName=ms-python.mypy-type-checker
+                "ms-python.mypy-type-checker",
+                // https://marketplace.visualstudio.com/items?itemName=ms-python.pylint
+                "ms-python.pylint",
+                // https://marketplace.visualstudio.com/items?itemName=ms-python.python
+                "ms-python.python",
+                // https://marketplace.visualstudio.com/items?itemName=ms-toolsai.datawrangler
```

Comment:
is this a data visualizer? what are its features?

Reply:
These are vscode extensions providing pylint and python support in vscode